Trimble.Modus.Components/Controls/TMSegmentedControl/TMSegmentedItem.xaml.cs

```diff
@@ -183,10 +207,17 @@ void UpdateCurrentItemStyle()
     /// </summary>
     void UpdateBackgroundColor()
     {
-        CurrentBackgroundColor = !IsSelected
-            ? ResourcesDictionary.ColorsDictionary(ColorsConstants.Transparent)
-            : SelectedBackgroundColor
-                ?? ResourcesDictionary.ColorsDictionary(ColorsConstants.TrimbleBlue);
+
+        string baseStyle = (ColorTheme == SegmentColorTheme.Primary) ? "Primary" : "Secondary";
```

Comment:
Can you try the VisualStateManager like in the other control for toggling b/w Normal and Selected?